common/app/mvt/MultiVariateTesting.scala

```diff
@@ -103,40 +90,5 @@ case class TestDefinition (
     sellByDate,
     exposeClientSide = true
   )
-
-  def isParticipating(implicit request: RequestHeader): Boolean = {
-    ActiveTests.getParticipatingTest(request).contains(this)
-  }
-}
-
-trait Tests {
-
-  protected def tests: Seq[TestDefinition]
-
-  def getParticipatingTest(request: RequestHeader): Option[TestDefinition] = {
-    getVariant(request).flatMap { variant =>
-      tests.find { test =>
-        test.variants.contains(variant) &&
-        test.switch.isSwitchedOn &&
-        ServerSideTests.isSwitchedOn
-      }
-    }
-  }
-
-  def isParticipatingInATest(request: RequestHeader): Boolean = getParticipatingTest(request).isDefined
-}
-
-object MultiVariateTesting {
-
-  sealed case class Variant(name: String)
-
-  private[mvt] val allVariants = List[Variant]()
-
-  def getVariant(request: RequestHeader): Option[Variant] = {
-    val cdnVariant: Option[String] = request.headers.get("X-GU-mvt-variant")
-
-    cdnVariant.flatMap( variantName => {
-      allVariants.find(_.name == variantName)
-    })
-  }
+  def isParticipating(implicit request: RequestHeader): Boolean = switch.isSwitchedOn && ServerSideTests.isSwitchedOn
```

Comment:
overriding is not mandatory. If you want to run an a/b test everywhere, you won't need to override isParticipating.